### ISSUE TITLE
App is missing....

### ISSUE THREAD

**Issue by kennymb5384** [Kemmey/Kemmey-TeslaWatch-Public#212]
After upgrading to the iPhone 12, the app is gone.  I cannot see it on the watch, the watch app, or my phone.  If I go into the App Store, it shows purchased, but I don't do anything.  I reached out to Apple, and the responded pretty quick, but said it was not on their end.

**Comment by Kemmey** [comment]
Hey @kennymb5384 

It IS on their end, unfortunately. And others have had luck getting Apple to fix it. I'm trying to get Apple to fix it for all, but they really don't give a shit :-(

In the mean time, you can always reinstall only the watch version directly from the watch itself: 

https://github.com/Kemmey/Kemmey-TeslaWatch-Public/issues/5#issuecomment-658742571

Try to reboot your phone, update to 14.2 if not already. Those are things that others have reported "fixing" the App Store showing purchased instead of allowing download.

**Comment by kennymb5384** [comment]
I was actually able to fix it.  I don't think I ever realized you could browse the App Store on the watch itself.  Everything on the phone showed 'purchased', but when I went to the App Store on the watch it had the cloud.  After I downloaded it from the watch, the phone all switched to the cloud on the App Store via the iPhone.  Everything is back to good!!!

**Comment by Kemmey** [comment]
@kennymb5384 That is fantastic, thank you for the feedback. This will certainly help others. So maybe "all" it takes is a complete uninstall, then reinstall from watch, then iOS store resets somehow, and then download from iOS. 

I'll send this along to Apple for them to look into. This cannot be expected behavior from apples side. 

Cheers,

Kim

**Comment by kennymb5384** [comment]
Yeah, I didn't even have to uninstall anything.  For some reason the cloud icon was just there in the watch app.  Phone app always showed purchased, but as soon as I downloaded form the watch, the phone app switched from purchased to the cloud icon!

**Comment by elbow007** [comment]
Unfortunately, this has not solved the problem for me so far. I’ve tried to delete the app both directly on the watch and from the watch app on iphone, and reinstalled from the watch app store, but the link in the iphone app store is still not clickable. Any ideas?
